mailer.rst

```diff
@@ -227,9 +227,8 @@ A round-robin transport is configured with two or more transports and the
 
     MAILER_DSN="roundrobin(postmark+api://ID@default sendgrid+smtp://KEY@default)"
 
-The mailer will start using a randomly selected transport and if it fails, it
-will retry the same delivery with the next transports until one of them succeeds
-(or until all of them fail).
+The round-robin transport is similar to the failover transport (see above), but
+it starts with a *randomly* selected transport instead of the first in the list.
```

Comment:
The way round-robin works: it starts with a randomly chosen transport, and then, for each email delivery, it chooses the next non-dead transport. So, for each delivery, the transport is different (except if there is only one non-dead transport left).

Reply:
Oh, right, thanks. I'll update and try to highlight the differences and similarities between round-robin and failover.